core/dwc_occurrence_2020-10-24.xml

```diff
@@ -0,0 +1,205 @@
+    <property group='Occurrence' name='sex' namespace='http://rs.tdwg.org/dwc/terms/' qualName='http://rs.tdwg.org/dwc/terms/sex' dc:relation='http://rs.tdwg.org/dwc/terms/index.htm#sex' dc:description='The sex of the biological individual(s) represented in the Occurrence. Recommended best practice is to use a controlled vocabulary.' examples='&quot;female&quot;, &quot;hermaphrodite&quot;, &quot;male&quot;' required='false'/>
+    <property group='Occurrence' name='lifeStage' namespace='http://rs.tdwg.org/dwc/terms/' qualName='http://rs.tdwg.org/dwc/terms/lifeStage' dc:relation='http://rs.tdwg.org/dwc/terms/index.htm#lifeStage' dc:description='The age class or life stage of the biological individual(s) at the time the Occurrence was recorded. Recommended best practice is to use a controlled vocabulary.' examples='&quot;egg&quot;, &quot;eft&quot;, &quot;juvenile&quot;, &quot;adult&quot;, &quot;2 adults 4 juveniles&quot;' required='false'/>
+    <property group='Occurrence' name='reproductiveCondition' namespace='http://rs.tdwg.org/dwc/terms/' qualName='http://rs.tdwg.org/dwc/terms/reproductiveCondition' dc:relation='http://rs.tdwg.org/dwc/terms/index.htm#reproductiveCondition' dc:description='The reproductive condition of the biological individual(s) represented in the Occurrence. Recommended best practice is to use a controlled vocabulary.' examples='&quot;non-reproductive&quot;, &quot;pregnant&quot;, &quot;in bloom&quot;, &quot;fruit-bearing&quot;' required='false'/>
+    <property group='Occurrence' name='behavior' namespace='http://rs.tdwg.org/dwc/terms/' qualName='http://rs.tdwg.org/dwc/terms/behavior' dc:relation='http://rs.tdwg.org/dwc/terms/index.htm#behavior' dc:description='A description of the behavior shown by the subject at the time the Occurrence was recorded.  Recommended best practice is to use a controlled vocabulary.' examples='&quot;roosting&quot;, &quot;foraging&quot;, &quot;running&quot;' required='false'/>
+    <property group='Occurrence' name='establishmentMeans' namespace='http://rs.tdwg.org/dwc/terms/' qualName='http://rs.tdwg.org/dwc/terms/establishmentMeans' dc:relation='http://rs.tdwg.org/dwc/terms/index.htm#establishmentMeans' dc:description='Statement about whether an organism or organisms have been introduced to a given place and time through the direct or indirect activity of modern humans. Recommended best practice is to use controlled value strings from the http://rs.tdwg.org/dwcem/ controlled vocabulary designated for use with this term. For details, refer to https://doi.org/10.3897/biss.3.38084' examples='&quot;native&quot;, &quot;nativeReintroduced&quot;, &quot;introduced&quot;, &quot;introducedAssistedColonisation&quot;, &quot;vagrant&quot;, &quot;uncertain&quot;' required='false'/>
```

Comment:
I wonder if we should link to a thesaurus here, cf. `occurrenceStatus`

    thesaurus="http://rs.gbif.org/vocabulary/gbif/establishment_means.xml

Probably updated to `http://rs.gbif.org/vocabulary/gbif/establishment_means_2020-10-26.xml` (to be created)